features/ubuntu_pro.feature

```diff
@@ -613,18 +613,11 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
     @uses.config.machine_type.gcp.pro
     @uses.config.machine_type.aws.pro
     @uses.config.machine_type.azure.pro
-    Scenario Outline: Auto-attach no-op when cloud-init has ubuntu_advantage on userdata
-        Given a `<release>` machine with ubuntu-advantage-tools installed adding this cloud-init user_data:
-        # This user_data should not do anything, just guarantee that the ua-auto-attach service
-        # # does nothing
-        """
-        ubuntu_advantage:
-        """
-        When I run `cloud-init query userdata` with sudo
-        Then stdout matches regexp:
-        """
-        ubuntu_advantage:
-        """
+    Scenario Outline: Auto-attach no-op when cloud-init has ubuntu_advantage on userdata and cloud-init auto attaches
+        # In the first boot we install uat and daily cloud-init and prepare/check preconditions
+        # In the second boot we execute the test
+
+        Given a `<release>` machine with ubuntu-advantage-tools installed and cloud-init upgraded to the latest daily version
```

Comment:
I understand that right now we need to use the daily version of cloud-init to test that, but once it is released, I think we should keep using the one in the archive instead, otherwise this test could be flaky, since we will be keep using the devel version of cloud-init.

I think we can add this test as is, but we need to update it after the release of cloud-init that possess that ua attach feature